builder/manifest.json

```diff
@@ -602,6 +609,76 @@
         ]
       }
     },
+    {
+      "id": "dashboardv2-base",
+      "base": "go",
+      "layers": [{
+        "script": "builder/img/dashboardv2-base.sh",
+        "inputs": [
+          "controller/api/*.proto"
```

Comment:
Is this input required for this layer?

Reply:
It shouldn't be. I think that's probably remanence of an earlier configuration.